.github/workflows/darker.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - main
-      - development
+      - v*
```

Comment:
Did you want to do `v*.*.*`? Otherwise any branch starting with `v` will match...

Reply:
tbh I like the simplicity and we can likely avoid other types of upstream branches that start with "v".

Reply:
updated to use `v*.*.*` ✔️